src/main/java/mod/gate/Gate.java

```diff
@@ -1,17 +1,13 @@
 package mod.gate;
 
 import com.google.gson.JsonParseException;
-import com.mojang.blaze3d.systems.RenderSystem;
-import mod.gate.config.Configuration;
-import mod.gate.events.EventHandler;
+import mod.gate.core.config.Configuration;
+import mod.gate.core.events.EventHandler;
+import mod.gate.utils.*;
```

Comment:
No wild card imports please

Reply:
didn't even know I added a wildcard import, so my guess is intelliJ did it